components/keys/Cargo.toml

```diff
@@ -17,12 +17,18 @@ rand = "0.7"
 failure = "0.1.5"
 pwbox = "0.2.1"
 secret-tree = "0.2.0"
-serde = "1.0.10"
-serde_derive = "1.0.64"
+serde = "1.0"
+serde_json = "1.0"
+serde_derive = "1.0"
+structopt = "0.3"
 toml = "0.5.0"
 
 exonum-crypto = { version = "1.0.0-rc.1", path = "../crypto" }
 
 [dev-dependencies]
 tempdir = "0.3.7"
 hex = "=0.4.0" # 0.4.1 is not compatible with Rust 1.36
+
+[[example]]
```

Comment:
Nit: Aren't examples added automatically by Cargo?

Reply:
Yes, they are. But I just prefer binary files with a dash as a splitter rather than an underscore 🙃

Reply:
And one more reason for adding this section. Without this section there is no possibility to run tests from the example. We should add  a `test=true` explicitly in this section to say to the cargo that we want to run tests from the example.

Reply:
Ah, OK; these are good reasons to have the example declared explicitly.